SpectroscopyBModeUI.cxx

```diff
@@ -118,6 +118,7 @@ void SpectroscopyBModeUI::ConnectProbe()
     ui->layout_Frequency->addWidget( freqCheckBoxes[ i ], 1);
     }
   ui->dropDown_Frequency->setCurrentIndex(0);
+  intersonDevice.SetFrequency( 0 );
```

Comment:
I thought changing the ui->dropDown_Frequency will trigger an event that sets the frequency